samples/cpp/cloning_gui.cpp

```diff
@@ -39,7 +39,11 @@
 #include <iostream>
 #include <stdlib.h>
 
-using namespace std;
+using std::cin;
```

Comment:
Could you please add some one-line comment about avoiding using of `using namespace std;` which cause conflict in C++17 (to prevent future refactoring back to "using namespace std")

Reply:
sure.